Tasks/UsePythonVersion/Tests/package.json

```diff
@@ -19,6 +19,8 @@
   "devDependencies": {
     "@types/mocha": "^2.2.48",
     "@types/mockery": "^1.4.29",
```

Comment:
Curious, why do the typing versions not match the package versions?

Reply:
It looks like the `@types` packages are versioned following their corresponding JS packages, but these must be lagging.  They're not necessarily maintained by the authors of the JS packages.  I haven't noticed any issues.

(The old "typings" files that were checked into source control have been deprecated in favor of just using NPM packages, which is what these `@types` packages are.)